<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Keyword:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item row="2" column="2" rowspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Find</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>&lt;&lt; Add</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Remove &gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Your Text:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QListWidget" name="OutPut"/>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="GraphWidget" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QGraphicsView</extends>
   <header>graphwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
